pkg/util/etcdutil/migrate.go

```diff
@@ -62,6 +64,11 @@ func Migrate(kubeConfig clientcmd.ClientConfig, etcdServiceIP string) error {
 	}
 	glog.Info("etcd cluster for migration is now running")
 
+	etcdServiceIP, err := getServiceIP(kubecli, api.NamespaceSystem, asset.EtcdServiceName)
```

Comment:
if this is a simple variable declaration, then yes. but this is a func call that does some initializations.